ihontaryk/homework_2/multiply.py

```diff
@@ -0,0 +1,13 @@
+from numbers import Number
+
+
+def multiply_numbers(number1, number2):
+    """
+    multiply_numbers function
+    """
+
+    if not (isinstance(number1, Number) and isinstance(number2, Number)):
+        raise TypeError(f"You used wrong type: expected int or float for both arguments")
```

Comment:
I do not like complex expressions especially with 'not' key, so usually I suggest to segregate condition to variable:

```
both_numbers = isinstance(number1, Number) and isinstance(number2, Number)
if not both_numbers:
    ...

```